editor/import/resource_importer_csv_translation.cpp

```diff
@@ -54,7 +54,7 @@ String ResourceImporterCSVTranslation::get_save_extension() const {
 
 String ResourceImporterCSVTranslation::get_resource_type() const {
 
-	return "StreamCSVTranslation";
+	return "Translation";
```

Comment:
Not sure about this one, it looks a bit dangerous since not all translations are CSV.